includes/definitions/discovery/ucopia.yaml

```diff
@@ -12,9 +12,8 @@ modules:
                 -
                     oid: highAvailabilityStatus
                     value: highAvailabilityStatus
-                    num_oid: '.1.3.6.1.4.1.31218.3.7.0.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.31218.3.7.0'
```

Comment:
Hi @mathieu-artic 
Why don't you do `num_oid: '.1.3.6.1.4.1.31218.3.7.{{ $index }}`. Index will always be "0" (which is fine if only one highAvailabilityStatus is provided.

Reply:
(plus simple en français)
pour une raison que je ne connais pas, LibreNMS n'a pas l'air d'interpréter de la même façon
on peut le voir sur le screenshot d'hier (#13031) il met "OK" au lieu de ce qui est indiqué dans le YAML
Pire, pour "Debug Value" ce n'est pas le bon chiffre qui est interprété et il affiche "Emergency" à la place de "Notice"
J'ai vérifié dans le débug, le nombre que LibreNMS reçoit est le bon

Reply:
Plus simple pour nous, mais pas forcément pour les autres :) 

My suggestion is to remove the "0" : 
from 
`                    num_oid: '.1.3.6.1.4.1.31218.3.7.0.{{ $index }}'`
to
`                    num_oid: '.1.3.6.1.4.1.31218.3.7.{{ $index }}'`

Could you try this variant ?

Reply:
Good idea ! I will try this

Reply:
i'ts work great, and all tests passed, thank's !